debian/rules

```diff
@@ -31,6 +31,11 @@ ifeq ($(DEB_HOST_ARCH),s390x)
 	COMMON_CONFIGURE_OPTIONS += -DMIR_LINK_TIME_OPTIMIZATION=OFF
 endif
 
+# Disable Mir EGL on Ubuntu 18.04 even if mesa has "Mir EGL" patch
+ifneq ($(filter buster/sid,$(call dpkg_late_eval,DEBIAN_VERSION,cat /etc/debian_version)),)
```

Comment:
I think it would be better to use `lsb_release --codename` here, and switch on `bionic`? This is correct, but surprised me.

(Not blocking)